jina/clients/request.py

```diff
@@ -18,12 +18,10 @@
                                      Tuple[DocumentSourceType, DocumentSourceType]]]
 
 
-def _build_doc(data, data_type: DataInputType, override_doc_id, **kwargs) -> Tuple['Document', 'DataInputType']:
+def _build_doc(data, data_type: DataInputType, **kwargs) -> Tuple['Document', 'DataInputType']:
```

Comment:
why don't we allow override_doc_id so that the user can choose that we give for them a `random id? Have we checked if some example uses this?

Reply:
I don't get the question. Either, the user gives and `id` in the data or the `Document` gets a random `id`. In either case, the user can decide to overwrite the `id` later via `doc.id = 1337`.

Reply:
ok good, using this `_build_doc` will generate random ids, if the user wants custom ids, they have to use their own generator. Fair enough